diff --git a/node_modules/cli-table/lib/index.js b/node_modules/cli-table/lib/index.js
index 80628e3..1a407ab 100644
--- a/node_modules/cli-table/lib/index.js
+++ b/node_modules/cli-table/lib/index.js
@@ -127,9 +127,9 @@ Table.prototype.toString = function (){
   };
 
   function get_width(obj) {
-    return typeof obj == 'object' && obj.width != undefined
+    return typeof obj == 'object' && obj && obj.width != undefined
          ? obj.width
-         : ((typeof obj == 'object' ? utils.strlen(obj.text) : utils.strlen(obj)) + (style['padding-left'] || 0) + (style['padding-right'] || 0))
+         : ((typeof obj == 'object' && obj !== null ? utils.strlen(obj.text) : utils.strlen(obj)) + (style['padding-left'] || 0) + (style['padding-right'] || 0))
   }
 
   // draws a line
@@ -179,7 +179,7 @@ Table.prototype.toString = function (){
 
     // transform array of item strings into structure of cells
     items.forEach(function (item, i) {
-      var contents = item.toString().split("\n").reduce(function (memo, l) {
+      var contents = (item == null ? '' : item).toString().split("\n").reduce(function (memo, l) {
         memo.push(string(l, i));
         return memo;
       }, [])
